src/jdk.jdwp.agent/share/native/libdt_socket/socketTransport.c

```diff
@@ -740,11 +740,6 @@ socketTransport_startListening(jdwpTransportEnv* env, const char* address,
         listenAddr = &(addrInfo[0]);
     }
 
-    if (listenAddr == NULL) {
```

Comment:
There's a typo in the comment of the original change:

`// No address of preferred address family found, grab the fist one.`

nit typo: s/fist/first/